packages/server/src/api/routes/tests/rowAction.spec.ts

```diff
@@ -0,0 +1,314 @@
+import _ from "lodash"
+import tk from "timekeeper"
+
+import { CreateRowActionRequest, RowActionResponse } from "@budibase/types"
+import * as setup from "./utilities"
+import { generator } from "@budibase/backend-core/tests"
+import { Expectations } from "src/tests/utilities/api/base"
+
+describe("/rowsActions", () => {
+  const config = setup.getConfig()
+
+  let tableId: string
+
+  beforeAll(async () => {
+    tk.freeze(new Date())
+    await config.init()
+  })
+
+  beforeEach(async () => {
+    const table = await config.api.table.save(setup.structures.basicTable())
+    tableId = table._id!
+  })
+
+  afterAll(setup.afterAll)
+
+  async function createRowAction(
+    tableId: string,
+    rowAction: CreateRowActionRequest,
+    expectations?: Expectations,
+    opts?: { publicUser?: boolean }
+  ) {
+    return await config.api.rowAction.save(
+      tableId,
+      rowAction,
+      {
+        ...expectations,
+        status: expectations?.status || 201,
```

Comment:
I'd probably put this default in `config.api.rowAction.save` instead of having this helper for it. I've done similar in other endpoints that have a non-200 success status.